notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -220,6 +220,56 @@ father and son [Sherman Klump had a heart to heart talk with his father Cletus a
 giant hamster []
 
 
+movie: Hollow Man (2000)
+========================
+
+:: Title
+Hollow Man
+
+:: Description
+Hollow Man is a 2000 science fiction horror film directed by Paul Verhoeven
+and starring Kevin Bacon, Elisabeth Shue and Josh Brolin.
+
+Synopsis: A scientist volunteers to be the first human test subject for a
+serum that renders the user invisible. When his fellow scientists are unable
+to restore him back to normal, he eventually becomes violently insane and goes
+on a murderous rampage.
+
+:: Date
+2000-08-04
+
+:: References
+https://en.wikipedia.org/wiki/Hollow_Man
+
+:: Ratings
+3 <paul>
+
+:: Choice Themes
+personal invisibility cloak [A main novelty of the film involves the scientist Sebastian Caine injecting himself with an experimental serum that renders its subject invisible and its unfortunate aftermath.]
+invisibility ability [A main novelty of the film surrounds the scientist Sebastian Caine's experience of what it is like to be invisible.]
+
+:: Major Themes
+the human capacity for good and evil [Central to the story was the already narcissistic scientist Sebastian Caine losing all sense of morality once he became invisible and didn't need to worry about facing any consequences for his actions anymore.]
+romantic jealousy [Sebastian became increasingly consumed with jealousy after finding out that his colleague Matt was carrying on with Sebastian's old flame, Linda, behind his back.]
+old flames [Sebastian made continued efforts to rekindle his relationship with his old flame and current colleague, Linda, but she had already moved on.]
+office romance [The co-workers at the lab Linda and Matt were having a relationship in secret behind the lab principal investigator Sebastian's back.]
+romantic love [The main characters Linda and Matt were head over heels in love.]
+narcissism [The scientist Sebastian Caine had an over-inflated ego, routinely boasted to others about how much of a genius he was, and even referred to himself as God on two separate occasions.]
```

Comment:
Sounds like arrogance, attitude of superiority and hubris.

Reply:
I think narcissism is typically when a character is so in love with themself that other people don't matter much. Witness Narcissus or Cat admiring their own reflection for excessive amounts of time.

Reply:
If you have to go out of your way in order to inform other people of your superiority then it is quite a different mental state. That reeks of insecurity and concern for social status, which is not what narcissism is about.

Reply:
I changed **narcissism** to **attitude of superiority**, but a plain reading of the **narcissism** definition suggests to me that it applies to him. You're adding criteria that are not made mention of in the definition.